spec/lib/video_info/providers/vimeo_spec.rb

```diff
@@ -215,4 +220,11 @@
     its(:video_id) { should eq '111431415' }
   end
 
+  context "with video 126641548 in /user*/review/126641548/* url", :vcr do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.